nemo/collections/llm/tools/auto_configurator/core/training_config.py

```diff
@@ -550,13 +585,13 @@ class T5GridSearch:
     gpu_memory_gb: int
     valid_pp: List[int]
 
-    tp = [1, 2, 4, 8]
-    pp = [1]
-    cp = [None]
-    ep = [None]
-    mbs = [1, 2, 4, 6, 8, 12, 16]
+    tp: List[int] = field(default_factory=lambda: [1, 2, 4, 8])
+    pp: List[int] = field(default_factory=lambda: [1])
+    cp: List[int] = field(default_factory=lambda: [None])
+    ep: List[int] = field(default_factory=lambda: [None])
+    mbs: List[int] = field(default_factory=lambda: [1, 2, 4, 6, 8, 12, 16])
 
-    gbs: int = 1920
+    gbs: List[int] = field(default_factory=lambda: [1920])
```

Comment:
Same as above - do these need to be `lamdba` functions or can they just be standard defaults?